main/activity-log.js

```diff
@@ -0,0 +1,79 @@
+'use strict'
+
+const Store = require('electron-store')
+const configStore = new Store({
+  name: 'activity-log'
+})
+
+class ActivityLog {
+  #entries
+  #lastId
+
+  constructor () {
+    this.#entries = loadStoredEntries()
+    this.#lastId = this.#entries
+      // We are storing ids as strings to allow us to switch to GUIDs in the future if needed
+      // When looking for the max id used, we need to cast strings to numbers.
+      .map(e => +e.id)
```

Comment:
```suggestion
      .map(e => Number(e.id))
```

It's longer but I find this conveys the intention more clearly

Reply:
A long time ago, in the Crankshaft days, `+val` was much more performant than `Number(val)`.  I am not sure if it's still the case.

Anyhow, this line is not on a critical path; I am okay with preferring readability over performance.